### PR TITLE
Update the Eclipse Configuration Instructions

### PR DESCRIPTION
**Pull request by Sadaf91**

### CODE REVIEW

File: docs/manual/troubleshooting.tex [hunk 1]
```diff
@@ -1038,6 +1038,15 @@ \subsection{Build the Checker Framework Manual (this document)\label{building-ma
 \end{enumerate}
 
 
+\subsection{Configure Eclipse to edit the Checker Framework\label{building-eclipse}}
+\begin{enumerate}
+\item Download Eclipse from the \href{https://www.eclipse.org/downloads}{official Eclipse website} and install it.
+\item Once the installation is completed, run Eclipse and set the workspace to \$JSR308.
+\item Enter the main Eclipse working screen and under "File" tab, select Import, General, and then Existing Projects into workspace. After the Import Projects window appears, select "Select Root Directory", browse for \$JSR308 folder, and select the whole folder. Try to deselect the projects related to Eclipse, encryption, tutorial, and demo.
```
Comment (mernst) [inline docs/manual/troubleshooting.tex]: Why "try to deselect"?  Might the deselection fail?

File: docs/manual/troubleshooting.tex [hunk 2]
```diff
@@ -1038,6 +1038,15 @@ \subsection{Build the Checker Framework Manual (this document)\label{building-ma
...
+\item After all projects have been imported, everything must be built successfully.
```
Comment (mernst) [inline docs/manual/troubleshooting.tex]: Could you clarify this?  Does "must be" mean "the user should do something", or "Eclipse is guaranteed to have done something", or does it mean something else?

**Comment by wmdietl** [comment]
I made a few edits and am talking to Sadaf to get them into her branch.

**Comment by wmdietl** [comment]
@mernst Do you want to have another look?

File: docs/manual/troubleshooting.tex [hunk 3]
```diff
@@ -1073,6 +1073,30 @@ \subsection{Build the Checker Framework Manual (this document)\label{building-ma
...
 
+\subsection{Configure Eclipse to edit the Checker Framework\label{building-eclipse}}
+
+\begin{enumerate}
+\item Follow the instructions earlier in Section~\ref{build-source} to
+clone and build all projects from their sources.
+\item Download Eclipse from
+the \href{https://www.eclipse.org/downloads}{official Eclipse website}
+and install it.
+\item Run Eclipse and set the workspace to \code{\$JSR308}.
```
Comment (mernst) [inline docs/manual/troubleshooting.tex]: Is this the literal 7 characters `$JSR308`, or the value of that environment variable?  (Similar question below.)

File: docs/manual/troubleshooting.tex [hunk 4]
```diff
@@ -1073,6 +1073,31 @@ \subsection{Build the Checker Framework Manual (this document)\label{building-ma
...
+\item Download Eclipse from
+the \href{https://www.eclipse.org/downloads}{official Eclipse website}
+and install it.
+\item Run Eclipse and set the
+workspace to the directory pointed to by \code{\$JSR308}.
+\item Enter the main Eclipse working screen and in the ``File'' menu,
+select ``Import'' $\rightarrow$ ``General'' $\rightarrow$ ``Existing
+Projects into workspace''.
+\item After the ``Import Projects'' window appears, select ``Select
+Root Directory'', and browse for \code{\$JSR308} folder.
```
Comment (wmdietl) [inline docs/manual/troubleshooting.tex]: `browse for \code{\$JSR308} folder.` ->
`select the directory pointed to by \code{\$JSR308}.`

Comment (wmdietl) [reply docs/manual/troubleshooting.tex]: @Sadaf91 Can you make this slight wording change?

**Pull request merged**